bugbug/models/regressor.py

```diff
@@ -110,6 +110,15 @@ def get_labels(self):
             )
         )
 
+        self.feature_extractors += [
```

Comment:
The reponsibility of the `get_labels` function is to only get the labels, we shouldn't overload it to do unrelated things.

What we want to add here is a kind of `fit`, so we should probably do it in `CommitExtractor::fit` (that is, what you are currently doing in files::__init__ you can do in files::fit).